lib/msf/core/payload/uuid/options.rb

```diff
@@ -116,7 +116,7 @@ def record_payload_uuid_url(uuid, url)
     payload_info = {
         uuid: uuid.puid_hex,
     }
-    payload = framework.db.get_payload(payload_info)
+    payload = framework.db.payloads(payload_info).first
```

Comment:
This is was previously specifically changed due to impacts around `workspace` interactions when calling `framework.db.payloads(payload_info).first` another pattern or further investigation is needed to avoid this pattern.